generate_images_using_openai_and_store_in_google_drive/main.bal

```diff
@@ -16,16 +16,19 @@ public function main() returns error? {
 
     sheets:Column range = check gSheets->getColumn(sheetId, sheetName, "A");
     foreach var cell in range.values {
-        string prompt = check cell.ensureType();
+        string prompt = cell.toString();
         images:CreateImageRequest imagePrompt = {
             prompt,
             response_format: "b64_json"
         };
         images:ImagesResponse imageRes = check openAIImages->/images/generations.post(imagePrompt);
-        string encodedImage = check imageRes.data[0].b64_json.ensureType();
-
-        // Decode the Base64 string and store image in Google Drive
-        byte[] imageBytes = check array:fromBase64(encodedImage);
-        _ = check gDrive->uploadFileUsingByteArray(imageBytes, string `${prompt}.png`, gDriveFolderId);
+        string? encodedImage = imageRes.data[0].b64_json;
+        if encodedImage is string {
+            // Decode the Base64 string and store image in Google Drive
+            byte[] imageBytes = check array:fromBase64(encodedImage);
+            _ = check gDrive->uploadFileUsingByteArray(imageBytes, string `${cell}.png`, gDriveFolderId);
+        } else {
+            return error(string `Failed to generate image for prompt: ${prompt}`);
+        }
```

Comment:
check whether is it the best practices to avoid having the else

Reply:
Yes, use early return on error case and avoid else

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/59/commits/a3bb18b9f59a83acd7f228caba3b1a4f59c841d4.